src/query-client/api/role.js

```diff
@@ -35,3 +35,10 @@ export const getRolesPermissions = async ({ ids }, config) => {
   }
   return [];
 };
+
+export const getMembersRoles = async ({ ids }, config) => {
+  if (ids.size) {
+    return Promise.all(ids.map((id) => getMemberRoles({ id }, config)));
```

Comment:
If you remove `async` you won't need `Promise.all`.

Reply:
And you won't need a special case with `return []` as long as the `size` is defined (size = 0 shouldn't throw an error).